packages/apollo/src/commands/service/list.ts

```diff
@@ -0,0 +1,168 @@
+import { flags } from "@oclif/command";
+import { ProjectCommand } from "../../Command";
+import { GraphQLSchema } from "graphql";
+import sortBy from "lodash.sortby";
+import { table } from "heroku-cli-util";
+import moment from "moment";
+import { ApolloConfig, isServiceProject } from "apollo-language-server";
+import chalk from "chalk";
+import {
+  ListServices_service_implementingServices,
+  ListServices_service_implementingServices_FederatedImplementingServices_services
+} from "apollo-language-server/lib/graphqlTypes";
+
+interface TasksOutput {
+  config: ApolloConfig;
+  implementingServices: ListServices_service_implementingServices | null;
+}
+
+const formatImplementingService = (
+  implementingService: ListServices_service_implementingServices_FederatedImplementingServices_services
+) => {
+  return {
+    name: implementingService.name,
+    url: implementingService.url || "",
+    updatedAt: `${moment(implementingService.updatedAt).format(
+      "D MMMM YYYY"
+    )} (${moment(implementingService.updatedAt).fromNow()})`
+  };
+};
+
+function formatHumanReadable({
+  implementingServices,
+  graphName,
+  frontendUrl
+}: {
+  implementingServices: ListServices_service_implementingServices | null;
+  graphName: string | undefined;
+  frontendUrl: string | undefined;
+}): string {
+  let result = "";
+  if (
+    !implementingServices ||
+    implementingServices.__typename === "NonFederatedImplementingService"
+  ) {
+    result =
+      "\nThis graph is not federated, there are no services composing the graph";
+  } else if (implementingServices.services.length === 0) {
+    result = "\nThere are no services on this federated graph";
+  } else {
+    // Create a sorted list of the services.
+    const sortedImplementingServices = sortBy<
+      ListServices_service_implementingServices_FederatedImplementingServices_services
+    >(implementingServices.services, [service => service.name.toUpperCase()]);
+
+    table(
+      [...sortedImplementingServices.map(formatImplementingService)].filter(
+        Boolean
+      ),
+      {
+        columns: [
+          { key: "name", label: "Name" },
+          { key: "url", label: "URL" },
+          { key: "updatedAt", label: "Last Updated At" }
+        ],
+        // Override `printHeader` so we don't print a header
+        printHeader: () => {},
+        // The default `printLine` will output to the console; we want to capture the output so we can test
+        // it.
+        printLine: line => {
+          result += `\n${line}`;
+        }
+      }
+    );
+    const serviceListUrlEnding = `/graph/${graphName}/service-list`;
+    const targetUrl = frontendUrl
+      ? `${frontendUrl}${serviceListUrlEnding}`
+      : `https://engine.apollographql.com${serviceListUrlEnding}`;
```

Comment:
Can we reference the constant / source of truth where this string actually lives? This way if we update one, they both change!

```js
export const DefaultEngineConfig = {
  endpoint: "https://engine-graphql.apollographql.com/api/graphql",
  frontend: "https://engine.apollographql.com"
};
```